io.catenax.request_for_quotation/2.0.0/RequestForQuotation.ttl

```diff
@@ -0,0 +1,577 @@
+
+:partName a samm:Property ;
+   samm:preferredName "part name"@en ;
+   samm:description "RFQ-PartConfiguration: name of the part in CAD file. In case the CAD file has only one part, this could also be the file name."@en ;
+   samm:characteristic :PartNameCharacteristic ;
+   samm:exampleValue "Drehteil" .
+
+:manufacturingDomain a samm:Property ;
+   samm:preferredName "manufacturing domain"@en ;
+   samm:description "RFQ-PartConfiguration: domain of required process (e.g. subtractive manufacturing, additive manufacturing, assembly)"@en ;
+   samm:characteristic :ManufacturingDomainCharacteristic ;
+   samm:exampleValue "additive manufacturing" .
+
+:billOfProcess a samm:Property ;
+   samm:preferredName "bill of process"@en ;
+   samm:description "RFQ-PartConfiguration: list of processes to manufacture the part(s)"@en ;
+   samm:characteristic :BillOfProcessCharacteristic .
+
+:processes a samm:Property ;
+   samm:preferredName "processes"@en ;
+   samm:description "RFQ-PartConfiguration: list of processes related to the manufacturing domain (subtractive Manufacturing: milling, drilling, surface treatment; additive manufacturing: powder bed fusion, heat treatment, surface finishing; assembly: welding,...).These process steps might also be hierarchical (milling:roughing, milling:finishing,..."@en ;
+   samm:characteristic :SetOfProcessStepsCharacteristic .
+
+:generalTolerance a samm:Property ;
+   samm:preferredName "general tolerance"@en ;
+   samm:description "RFQ-PartConfiguration: selection of defined tolerance classes"@en ;
+   samm:characteristic :GeneralToleranceCharacteristic ;
+   samm:exampleValue "ISO 2768-1 (m), Rough DIN ISO 2768 cL, Medium DIN ISO 2768 mK, Fine DIN ISO 2768 fH" .
+
+:additionalRequirements a samm:Property ;
+   samm:preferredName "additional requirements"@en ;
+   samm:description "RFQ-PartConfiguration: specific requirements e.g. to guarantee quality, like premium quality check or standard quality check or test part required (makes only sense for batch)"@en ;
+   samm:characteristic :AdditionalQualityRequirementsCharacteristic ;
+   samm:exampleValue "premium quality check, standard quality check" .
+
+:material a samm:Property ;
+   samm:preferredName "part material"@en ;
+   samm:description "RFQ-PartConfiguration: material definition with further information"@en ;
+   samm:characteristic :MaterialCharacteristic .
+
+:partQuantity a samm:Property ;
+   samm:preferredName "part quantity"@en ;
+   samm:description "RFQ-PartConfiguration: Quantity of which the part item is assembled into the parent item (within the CAD File). In general it is '1' for serialized parts."@en ;
+   samm:characteristic :PartQuantityCharacteristic .
+
+:quantityNumber a samm:Property ;
+   samm:preferredName "Quantity Number"@en ;
+   samm:description "The number of objects related to the measurement unit"@en ;
+   samm:characteristic :NumberOfObjects ;
+   samm:exampleValue "2.5"^^xsd:double .
+
+:measurementUnit a samm:Property ;
```

Comment:
`preferredName` missing

Reply:
Preferred name for `measurementUnit` has been added